### PR TITLE
Add a new atomic test to T1560.001.yaml

### PR DESCRIPTION
**Pull request by aranhams**
**Details:**

This pull request adds a new atomic test to the existing codebase, with a focus on using AES and base64 to encrypt compressed data and send it to a C2 server.
This atomic test tries to emulate the behaviour of the FLEXIROOT backdoor to archive the collected data. FLEXIROOT typically utilizes AES encryption and base64 encoding to transfer the encrypted data to the C2 server. 
In this test, standard zip compression and the OpenSSL library are used to encrypt the compressed data.

**Testing:**

I tested the commands on Linux and Mac operating systems.

**Associated Issues:**

### CODE REVIEW

File: atomics/T1560.001/T1560.001.yaml [hunk 1]
```diff
@@ -276,3 +276,41 @@ atomic_tests:
...
+      description: Password used to encrypt the data
+      type: string
+      default: atomic_enc_pass
+  dependency_executor_name: 
+  dependencies: 
+  executor:
+    command: |-
+      mkdir -p #{input_folder}
+      cd #{input_folder}
+      touch {a..z}.data
```
Comment (cyberbuff) [inline atomics/T1560.001/T1560.001.yaml:309]: Can you move the `folder` and `file` creation to dependencies ?

File: atomics/T1560.001/T1560.001.yaml [hunk 2]
```diff
@@ -276,3 +276,41 @@ atomic_tests:
...
+    input_file:
+      description: 'Name of the compressed and encrypted files'
+      type: String
+      default: t1560_data
+    enc_pass:
+      description: Password used to encrypt the data
+      type: string
+      default: atomic_enc_pass
+  dependency_executor_name: 
+  dependencies:
```
Comment (cyberbuff) [inline atomics/T1560.001/T1560.001.yaml:304]: You can remove these two lines as they are empty.

**Review by cyberbuff — CHANGES_REQUESTED** [review]
Thanks for your first-time contribution. Requested some minor changes here.

File: atomics/T1560.001/T1560.001.yaml [hunk 3]
```diff
@@ -276,3 +276,41 @@ atomic_tests:
       ls -l #{test_folder}
     cleanup_command: |
       rm -Rf #{test_folder}
+- name: 'Encrypts collected data with AES-256 and Base64'
+  auto_generated_guid:
```
Comment (cyberbuff) [inline atomics/T1560.001/T1560.001.yaml:280]: GUIDs will be auto generated. You can remove this one too.

File: atomics/T1560.001/T1560.001.yaml (hunk 1 above)
Comment (aranhams) [reply atomics/T1560.001/T1560.001.yaml]: Yep! I moved the `folder` and `file` creation to dependencies :)

File: atomics/T1560.001/T1560.001.yaml (hunk 2 above)
Comment (aranhams) [reply atomics/T1560.001/T1560.001.yaml]: As I moved the creation of `folder` and `file` to dependencies, the lines are no longer empty, so I kept them with the new commands.

File: atomics/T1560.001/T1560.001.yaml (hunk 3 above)
Comment (aranhams) [reply atomics/T1560.001/T1560.001.yaml]: Done! :)

**Comment by aranhams** [comment]
Hey @cyberbuff, thanks for the quick review of my pull request and for the suggestions. 
I added a new commit with your suggestions 😄 

Best,

File: atomics/T1560.001/T1560.001.yaml (hunk 2 above)
Comment (cyberbuff) [reply atomics/T1560.001/T1560.001.yaml]: Thank you

File: atomics/T1560.001/T1560.001.yaml (hunk 3 above)
Comment (cyberbuff) [reply atomics/T1560.001/T1560.001.yaml]: Thank you.

File: atomics/T1560.001/T1560.001.yaml [hunk 4]
```diff
@@ -276,3 +276,38 @@ atomic_tests:
...
+  - linux
+  - macos
+  input_arguments:
+    input_folder:
+      description: Path to the folder used to store the test files
+      type: String
+      default: /tmp/t1560
+    input_file:
+      description: 'Name of the compressed and encrypted files'
+      type: String
```
Comment (cyberbuff) [inline atomics/T1560.001/T1560.001.yaml:295]: Can you rename `String` to `string` in lines 291 and 295 to match the schema ?

Comment (aranhams) [reply atomics/T1560.001/T1560.001.yaml]: I made a typo in line 291, is a path and not a string.
Done, all fixed!

Comment (cyberbuff) [reply atomics/T1560.001/T1560.001.yaml]: Thank you

File: atomics/T1560.001/T1560.001.yaml [hunk 5]
```diff
@@ -276,3 +276,38 @@ atomic_tests:
...
+  dependency_executor_name: bash
+  dependencies:
+  - description: The folder and test files must exist
+    prereq_command: 'if [ ! -d #{input_folder} ]; then exit 1; else exit 0; fi;'
+    get_prereq_command: 'if [ ! -d #{input_folder} ]; then mkdir -p #{input_folder}; cd #{input_folder}; touch {a..z}.data; fi;'
+  executor:
+    command: |-
+      zip -r tmp.zip #{input_folder}
+      openssl enc -aes-256-cbc -pass pass:#{enc_pass} -p -in #{input_folder}/#{input_file}.zip -out #{input_folder}/#{input_file}.enc 
+      cat #{input_folder}/data.enc | base64
```
Comment (cyberbuff) [inline atomics/T1560.001/T1560.001.yaml:310]: One last change. Can you change `tmp.zip` and `data.enc` to use `input_file` parameter ?

**Review by cyberbuff — CHANGES_REQUESTED** [review]

File: atomics/T1560.001/T1560.001.yaml (hunk 5 above)
Comment (aranhams) [reply atomics/T1560.001/T1560.001.yaml]: Sorry, in the first version, I didn't use variables and when I change it I forgot to change static names.

Done!

Comment (cyberbuff) [reply atomics/T1560.001/T1560.001.yaml]: No worries. Thank you

**Review by cyberbuff — APPROVED** [review]

**Comment by cyberbuff** [comment]
Thanks for your first-time contribution. Keep it up! Don't forget to [claim your free t-shirt](https://github.com/redcanaryco/atomic-red-team/wiki/Contributing#claim-your-free-t-shirt)

**Pull request merged**